config/routes.rb

```diff
@@ -605,6 +605,7 @@ def call(env)
       end
 
       resources :active_docs, :controller => 'api_docs_services', except: %i[new edit]
+      resources :backend_api_active_docs, controller: 'api_docs_backend_apis', only: [:index, :update, :create, :destroy]
```

Comment:
I was wondering... can we have this as an actual resource of backend_apis, i.e. `/admin/api/backend_apis/{backend_api_id}/api_docs`?